src/V3LinkDot.cpp

```diff
@@ -1030,7 +1030,7 @@ class LinkDotFindVisitor final : public VNVisitor {
                 newvarp->funcReturn(true);
                 newvarp->trace(false);  // Not user visible
                 newvarp->attrIsolateAssign(nodep->attrIsolateAssign());
-                nodep->addFvarp(newvarp);
+                nodep->fvarp(newvarp);
```

Comment:
Separate and push now right to master?

Reply:
I think there will be more of these, so I will bundle them up before comitting